docs/consensus/security.md

```diff
@@ -13,4 +13,75 @@ keywords:
 - Constraining Simulation
 - Handling Equivocation
 - Detecting Long-Range Attacks -->
-For detailed security analysis please refer to our paper [Dilithium: A Proof-of-Archival-Storage Consensus Protocol for Subspace](https://github.com/subspace/consensus-v2-research-paper)
\ No newline at end of file
+
+A complex system like the Subspace protocol has many potential attack vectors, some are more general, blockchain related, 
+while other are focused on proof of capacity and specifically proof of archive storage, as used in Subspace.  
+
+This page attempts to give an overview of how such attacks are mitigated in the Subspace protocol. There are many things
```

Comment:
I'd say simply "gives an overview"

Reply:
Too apologetic?
Basically I'm trying to pass the message that the page should not be considered complete or comprehensive.

Reply:
You state that message clearly in the next sentence, so I think "gives an overview" is sufficient.

Reply:
I'll revise. Just to note that we should all aim for this page to be complete at some point.